webapp/security/fixtures/releases.py

```diff
@@ -0,0 +1,340 @@
+import sys
+from datetime import datetime
+
+sys.path.insert(0, "")
```

Comment:
Have you tried running it without this? I think using `dotrun` will avoid this problem.